Apps/AdminWebClient/src/Server/Controllers/EmailAdminController.cs

```diff
@@ -79,21 +80,28 @@ public IActionResult GetEmails()
         /// Creates a clone of the email id and requeues it for sending.
         /// </summary>
         /// <returns>An ok result unless the email id is not foun.</returns>
-        /// <param name="emailId">The email Id to resend.</param>
-        /// <response code="200">Returns the list of beta requests.</response>
+        /// <param name="emailIds">The email Ids to resend.</param>
+        /// <response code="200">Successfully queued all emails.</response>
         /// <response code="401">the client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
-        [HttpPost]
-        [Route("{emailId}")]
-        public IActionResult ResendEmail(Guid emailId)
+        [HttpPatch]
```

Comment:
I don't think this is appropriate as we are creating a net new resource and not updating something.

Reply:
+1